src/scenes/Login/Login.js

```diff
@@ -31,8 +36,10 @@ const Login = ({ history }) => {
       username,
       password
     })
-    if (foundUser) {
-      return history.replace('/')
+    console.log(getToken())
+    saveToken(foundUser)
+    if (getToken()) {
+      return window.location.replace('/')
```

Comment:
Why did you remove the old logic? Just check the found user, don't need to get the token then check the token. This takes more steps. Should be 👇 
submit form -> find user -> if(foundUser) -> saveToken and replace('/')

Reply:
Same at here, no need to `return`

Reply:
![image](https://user-images.githubusercontent.com/62316127/125916614-c11dd0b5-13db-4a2b-a8ae-a5059af7905f.png)